examples/ipv6/multicast/project-conf.h

```diff
@@ -44,7 +44,16 @@
 #include "net/ipv6/multicast/uip-mcast6-engines.h"
 
 /* Change this to switch engines. Engine codes in uip-mcast6-engines.h */
+#if WITH_SMRF
+#define UIP_MCAST6_CONF_ENGINE UIP_MCAST6_ENGINE_SMRF
+#define RPL_CONF_MOP 3
```

Comment:
Good catch...!

I think I hesitated to use `RPL_MOP_STORING_MULTICAST` because it's a *private* declaration in `rpl-private.h`, which is the file for "private declarations for ContikiRPL." Of course, it's by far better to use the macro instead of the immediate value.

Now I found `rpl-private.h` is included from outside of the `rpl` module. So, I guess I can do the same thing as others did, although it seems we don't need to include `rpl-private.h` within `project-conf.h`.